tests/test_tutorials.py

```diff
@@ -31,7 +34,13 @@ def test_tutorials():
         script_name = notebook_name[:-6] + '.py'
         os.system( 'ipython nbconvert --to=python %s' %notebook_name )
         clean_ipython_features( script_name )
-        execfile( script_name )
+        try:
+            exec( open(script_name).read() )
```

Comment:
Note: I am still *sourcing* the file which is probably not ideal and can have side-effects.

Using `os.system()` and checking its return status is probably the better choice for a follow-up refactoring.